core/services/job/testdata/pretty.toml

```
@@ -0,0 +1,66 @@
+
+[relayConfig.chainReader.chainContractReaders.median.chainReaderDefinitions.LatestRoundRequested]
+chainSpecificName = 'RoundRequested'
+readType = 1
```

Why do we use an obscure int enum value for the `ReadType`, instead of a string like:
```suggestion
readType = 'event'
```

---

sgtm, I can change this in a separate PR

---

wanted to change this at some point, but forgot, pushed a commit with changes